components/Report/models.rst

```diff
@@ -0,0 +1,73 @@
+The default interface contains the following attributes with appropriate setters and getters.
+
++---------------------------+----------------------------------------------------+
+| Attribute                 | Description                                        |
++===========================+====================================================+
+| id                        | Unique id of the report                            |
++---------------------------+----------------------------------------------------+
+| code                      | Unique code of the report                          |
++---------------------------+----------------------------------------------------+
+| name                      | Name of the report                                 |
++---------------------------+----------------------------------------------------+
+| description               | Description of your gorgeous report                |
++---------------------------+----------------------------------------------------+
+| renderer                  | Name of the renderer that visualize report data    |
++---------------------------+----------------------------------------------------+
+| rendererConfiguration     | Configuration of used renderer                     |
++---------------------------+----------------------------------------------------+
+| dataFetcher               | Name of the dataFetcher that provides report data  |
++---------------------------+----------------------------------------------------+
+| dataFetcherConfiguration  | Configuration of used data fetcher                 |
++---------------------------+----------------------------------------------------+
+
+.. note::
+
+    This model implements ``ReportInterface``
+
+
+The Data
+-----------
+
+Data is supporting model in SyliusReportBundle. That contains data provided by data fetcher and labels used to clarify report display.
+
++--------------+------------------------------------+
+| Attribute    | Description                        |
++==============+====================================+
+| labels       | Labels that describe fetched data  |
++--------------+------------------------------------+
+| data         | Data fetcher from database         |
++--------------+------------------------------------+
+
+
+DataFetcherInterface
+----------------------
+
+To characterize data fetcher object, its class needs to implement the ``DataFetcherInterface``.
+
++-------------------------------+---------------------------------------+
+| Method                        | Description                           |
++===============================+=======================================+
+| fetch(array $configuration)   | Fetch data from database              |
++-------------------------------+---------------------------------------+
+| getType()                     | Return type of data fetcher object    |
```

Comment:
We should stick to one form, IMO the description should be in the 3rd person (Returns, Renders etc.)